src/main/java/com/gtnewhorizons/angelica/glsm/GLStateManager.java

```diff
@@ -204,6 +204,9 @@ public static void reset() {
 
 
     public static class GLFeatureSet extends IntOpenHashSet {
+
+        private static final long serialVersionUID = 8558779940775721010L;
```

Comment:
What's this for?

Reply:
See https://stackoverflow.com/a/285809

Probably irrelevant for us, but Eclipse complained and it doesn't hurt to have it.